### ISSUE TITLE
突然无法登录

### ISSUE THREAD

**Issue by HateNoSaiShi** [upbit/pixivpy#85]
`    raise PixivError('[ERROR] auth() failed! check username and password.\nHTTP %s: %s' % (r.status_code, r.text), header=r.headers, body=r.text)

PixivError: [ERROR] auth() failed! check username and password.
HTTP 400: {"has_error":true,"errors":{"system":{"message":"access_denied","code":1508}}}`

**Comment by upbit** [comment]
Pixiv升级了auth接口，导致api登录失败，问题和#83重复了。可以先参考下面方法临时更新到master代码，pypi我会尽快提供新版本

```bash
pip uninstall pixivpy

git clone https://github.com/upbit/pixivpy.git
cd pixivpy/
pip install [--editable] .
```

**Comment by upbit** [comment]
Fixed, bump PyPi version to **3.4.0**. Please upgrade API using:

```bash
$ pip install pixivpy --upgrade

Collecting pixivpy
  Downloading https://files.pythonhosted.org/packages/a9/db/0e0b1ca48e184436289631ed2c6fd6313a93409c493c6af3830531780385/PixivPy-3.4.0-py2-none-any.whl
Requirement already satisfied, skipping upgrade: requests in /anaconda/lib/python2.7/site-packages (from pixivpy) (2.13.0)
Installing collected packages: pixivpy
Successfully installed pixivpy-3.4.0
```

**Comment by HateNoSaiShi** [comment]
thx
